docs/06_how-to-guides/40_multi-index/how-to-modify-data-in-a-multi-index-table.md

```diff
@@ -1,28 +1,69 @@
 ---
-content_title: How to modify data in a multi index table
+content_title: How to modify data in a multi-index table
 ---
 
-## Preconditions
-- It is assumed you already have a multi index table instance defined along with its mandatory primary index, otherwise take a look at the section [How to instantiate a multi index table](./how-to-instantiate-a-multi-index-table.md).
+## Overview
 
-To modify data in the multi index table defined in the above tutorial, you will implement an action `mod` which it will receive as parameter the `user` which is the key of the row you want to modify and the `value` param which is the value to update with the row.
+This guide provides instructions to modify data in a multi-index table.
+
+## Reference
+
+See the following code reference:
+
+* The [`multi-index`](../../classeosio_1_1multi__index) class.
+* The [`multi-index::modify(...)`](../../group__multiindex/#function-modify) method.
+
+## Before you begin
+
+Make sure you have the following prerequisites in place:
+
+* An EOSIO development environment, for details consult the [Get Started Guide](https://developers.eos.io/welcome/latest/getting-started-guide/index),
+* A multi-index `testab` table instance which stores `user` objects indexed by the primary key which is of type `eosio::name`. Consult the section [How to instantiate a multi-index table](./how-to-instantiate-a-multi-index-table) to learn how to set it up.
+
+## Procedure
+
+Complete the following steps to modify data in the `testtab` multi-index table.
+
+### 1. Define The mod(...) Action
+
+Add to the definition of the `testtab` multi-index table the `mod` action which gets as input parameters a `user` of type `eosio::name` and a `value` of type `uint32_t`. The `mod` action will update the `user` object `datum` data member with the `uint32_t` value.
+
+```cpp
+[[eosio::action]] void mod( name user, uint32_t value );
+```
+
+Optionally, for ease of use add the action wrapper definition as well.
+
+```diff
+[[eosio::action]] void mod( name user, uint32_t value );
+
++using mod_action = action_wrapper<"mod"_n, &multi_index_example::mod>;
+```
+
+### 2. Find The User You Want To Modify
+
+Make use of the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to modify. The targeted user is searched based on its account name.
```

Comment:
Make use of the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to modify. The targeted user is searched based on its account name.

=>



Use the multi-index [`find(...)`](../../group__multiindex#function-find) method to locate the user object you want to modify. The targeted user is searched based on its account name.